pkgs/by-name/ot/otb/itk_4_13/package.nix

```diff
@@ -0,0 +1,146 @@
+    "-DModule_ITKFiniteDifference=ON"
+    "-DModule_ITKGPUCommon=ON"
+    "-DModule_ITKGPUFiniteDifference=ON"
+    "-DModule_ITKImageAdaptors=ON"
+    "-DModule_ITKImageFunction=ON"
+    "-DModule_ITKMesh=ON"
+    "-DModule_ITKQuadEdgeMesh=ON"
+    "-DModule_ITKSpatialObjects=ON"
+    "-DModule_ITKTransform=ON"
+    "-DModule_ITKTransformFactory=ON"
+    "-DModule_ITKIOTransformBase=ON"
+    "-DModule_ITKIOTransformInsightLegacy=ON"
+    "-DModule_ITKIOTransformMatlab=ON"
+    "-DModule_ITKAnisotropicSmoothing=ON"
+    "-DModule_ITKAntiAlias=ON"
+    "-DModule_ITKBiasCorrection=ON"
+    "-DModule_ITKBinaryMathematicalMorphology=ON"
+    "-DModule_ITKColormap=ON"
+    "-DModule_ITKConvolution=ON"
+    "-DModule_ITKCurvatureFlow=ON"
+    "-DModule_ITKDeconvolution=ON"
+    "-DModule_ITKDenoising=ON"
+    "-DModule_ITKDisplacementField=ON"
+    "-DModule_ITKDistanceMap=ON"
+    "-DModule_ITKFastMarching=ON"
+    "-DModule_ITKFFT=ON"
+    "-DModule_ITKGPUAnisotropicSmoothing=ON"
+    "-DModule_ITKGPUImageFilterBase=ON"
+    "-DModule_ITKGPUSmoothing=ON"
+    "-DModule_ITKGPUThresholding=ON"
+    "-DModule_ITKImageCompare=ON"
+    "-DModule_ITKImageCompose=ON"
+    "-DModule_ITKImageFeature=ON"
+    "-DModule_ITKImageFilterBase=ON"
+    "-DModule_ITKImageFusion=ON"
+    "-DModule_ITKImageGradient=ON"
+    "-DModule_ITKImageGrid=ON"
+    "-DModule_ITKImageIntensity=ON"
+    "-DModule_ITKImageLabel=ON"
+    "-DModule_ITKImageSources=ON"
+    "-DModule_ITKImageStatistics=ON"
+    "-DModule_ITKLabelMap=ON"
+    "-DModule_ITKMathematicalMorphology=ON"
+    "-DModule_ITKPath=ON"
+    "-DModule_ITKQuadEdgeMeshFiltering=ON"
+    "-DModule_ITKSmoothing=ON"
+    "-DModule_ITKSpatialFunction=ON"
+    "-DModule_ITKThresholding=ON"
+    "-DModule_ITKEigen=ON"
+    "-DModule_ITKNarrowBand=ON"
+    "-DModule_ITKNeuralNetworks=ON"
+    "-DModule_ITKOptimizers=ON"
+    "-DModule_ITKOptimizersv4=ON"
+    "-DModule_ITKPolynomials=ON"
+    "-DModule_ITKStatistics=ON"
+    "-DModule_ITKRegistrationCommon=ON"
+    "-DModule_ITKGPURegistrationCommon=ON"
+    "-DModule_ITKGPUPDEDeformableRegistration=ON"
+    "-DModule_ITKMetricsv4=ON"
+    "-DModule_ITKPDEDeformableRegistration=ON"
+    "-DModule_ITKRegistrationMethodsv4=ON"
+    "-DModule_ITKClassifiers=ON"
+    "-DModule_ITKConnectedComponents=ON"
+    "-DModule_ITKDeformableMesh=ON"
+    "-DModule_ITKKLMRegionGrowing=ON"
+    "-DModule_ITKLabelVoting=ON"
+    "-DModule_ITKLevelSets=ON"
+    "-DModule_ITKLevelSetsv4=ON"
+    "-DModule_ITKMarkovRandomFieldsClassifiers=ON"
+    "-DModule_ITKRegionGrowing=ON"
+    "-DModule_ITKSignedDistanceFunction=ON"
+    "-DModule_ITKVoronoi=ON"
+    "-DModule_ITKWatersheds=ON"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    xz
+  ];
+  buildInputs = [ libuuid ];
+  propagatedBuildInputs = [
+    # similar to 5.2.x, we progagate these inputs for OTB
+    expat
+    fftw
+    fftwFloat
+    hdf5-cpp
+    libjpeg
+    libpng
+    libtiff
+    zlib
+  ];
+
+  meta = {
```

Comment:
set meta.platforms = linux or fix
```
In file included from /tmp/nix-build-itk-4.13.3.drv-0/source/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_sparse_matrix_linear_system.cxx:6:
In file included from /tmp/nix-build-itk-4.13.3.drv-0/source/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_sparse_matrix_linear_system.h:24:
/tmp/nix-build-itk-4.13.3.drv-0/source/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_sparse_matrix.h:97:29: error: no template named 'binary_function' in namespace 'std'; did you mean '__binary_function'?
  struct less : public std::binary_function<vnl_sparse_matrix_pair, vnl_sparse_matrix_pair, bool>
                       ~~~~~^~~~~~~~~~~~~~~
                            __binary_function
/nix/store/0m4n63p6lldjys591p2r5kbr87zcpfbz-libcxx-16.0.6-dev/include/c++/v1/__functional/binary_function.h:49:1: note: '__binary_function' declared here
using __binary_function = __binary_function_keep_layout_base<_Arg1, _Arg2, _Result>;
^
1 error generated.
```

Reply:
Thanks @FliegendeWurst !

I have set the platforms to Linux.